_pages/events/2024-07-12_hackathon-oxford.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: single
+classes: wide
+title: UNIVERSE-HPC Course Materials Hackathon
+permalink: /events/2024-07-12_hackathon-oxford/
+author_profile: false
+
+
+---
+
+**Join us for the second UNIVERSE-HPC Hackathon on Friday 12 July from 10:00 to 16:30 BST (UTC+1)**, which will take place **in person in Oxford and online**.
+
+### [Register now](https://forms.office.com/e/ysfdNKXaF2).
+
+## Overview:
+
+- 🕒 **Friday 12th April** 2024, **[10:00 - 16:30 BST](https://www.timeanddate.com/worldclock/fixedtime.html?msg=UNIVERSE-HPC+Course+Materials+Hackathon&iso=20240712T10&p1=1233&ah=6&am=30)**
+- 🌍 In person in Oxford and Online via Zoom
+
+Following up on our successful first Hackathon, the UNIVERSE-HPC project team is organising another Hackathon to continue working on enhancing our course materials. Anyone with an interest in training for RSEs is welcome - no need to have joined the previous Hackathon to join us for this one 💖
+
+We will focus our activity on two strands that you can get involved in and contribute to:
+
+- **Infrastructure**, enhancing/extending/contributing to Gutenberg, our new training platform. One of the likely foci is making Gutenberg more accessible.
+- **Pedagogy**, implementing good educational principles in the lessons. One of the likely activities will be to move over the lesson objectives we collated last time into the actual lessons.
+
+For people who did not attend the previous Hackathon, this will be a fantastic opportunity to learn about using and deploying Gutenberg, an open-source teaching platform developed at the University of Oxford, that you could use in your own teaching. Gutenberg uses course materials written in Markdown and offers the features of a learning management system, for example, giving access to a specific group of enrolled students and tracking their progress through a course. Have a look at an example deployment from Oxford: https://train.oxrse.uk/material.
```

Comment:
Add a link for the URL at the end of the paragraph:
```suggestion
For people who did not attend the previous Hackathon, this will be a fantastic opportunity to learn about using and deploying Gutenberg, an open-source teaching platform developed at the University of Oxford, that you could use in your own teaching. Gutenberg uses course materials written in Markdown and offers the features of a learning management system, for example, giving access to a specific group of enrolled students and tracking their progress through a course. Have a look at an example deployment from Oxford: [https://train.oxrse.uk/material](https://train.oxrse.uk/material).
```